cli/src/semgrep/commands/ci.py

```diff
@@ -145,6 +181,19 @@ def fix_head_if_github_action(metadata: GitMeta) -> Iterator[None]:
     is_flag=True,
     hidden=True,
 )
+@click.option(
+    "--fail-open",
+    "fail_open",
+    type=FAIL_OPEN_STATE_TYPE,
+    help="""
+        Configures how scan status is sent to the fail-open server.
+        If 'on', fail-open are always sent.
+        If 'off', fail-open are disabled altogether and not sent.
+        If absent, the SEMGREP_SEND_FAIL_OPEN environment variable value will be used.
+        If no environment variable, defaults to 'off'.
```

Comment:
Keep in mind this text shows up when someone runs `semgrep ci --help` so we should reword this with that in mind. E.g. even as a semgrep maintainer I don't understand what a "fail-open server" is and what it means if "fail-open are always sent".

Reply:
Reading through the code this looks like a "crash reporting server" and that "crash reports are always sent"?

Reply:
Updated in latest commit